acceptance/analyzer_test.go

```diff
@@ -781,13 +784,13 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 								ctrPath("/layers"),
 								analyzeImage,
 								h.WithFlags(
-									"--env", "CNB_REGISTRY_AUTH="+fixtures.authRegAuthConfig,
+									"--env", "CNB_REGISTRY_AUTH="+fixtures.regAuthConfig,
 									"--network", registryNetwork,
 									"--env", "CNB_PLATFORM_API="+platformAPI,
 								),
 								h.WithArgs(
 									ctrPath(analyzerPath),
-									fixtures.authRegAppImage,
+									fixtures.readWriteAppImage,
```

Comment:
Same comment as https://github.com/buildpacks/lifecycle/pull/685/files#r691403233 (but for app)